frontend/src/components/RecipeScraper.jsx

```diff
@@ -8,6 +8,16 @@ const RecipeScraper = ({
 }) => {
   const navigate = useNavigate();
 
+  // if the user is logged in - generate a recipe, else redirect to login page
+  const handleClick = async () => {
+    if (localStorage.getItem('token')) {
+      await handleScrapeRecipe();
```

Comment:
Is handleScrapeRecipe() being passed as props to RecipeScraper?
Hard to follow on github. Can't quite track where setToken is being used after getting back the token from handleScrapeRecipe. Though I guess the code atm doesn't return a token from the backend.

Reply:
Yeah, handleScrapeRecipe is being passed as a prop, not so sure about the token, I added to the backend a generate token for the generate recipe, but that hasn't been merged, before that I think the token wasn't really needed to scrape to recipe (could be wrong, but that was how we first did it)

Reply:
Yeah, It wasn't being used, but probably need to be.